test/basic.js

```diff
@@ -64,6 +64,36 @@ describe('request', function(){
         done();
       });
     })
+
+    it('is optional with a promise', function() {
```

Comment:
Ah, I haven't seen that before. Looks good then.

On Tue, Mar 8, 2016 at 9:17 AM Kornel <notifications@github.com> wrote:

> In test/basic.js
> <https://github.com/visionmedia/superagent/pull/925#discussion_r55382858>:
>
> > @@ -64,6 +64,36 @@ describe('request', function(){
> >          done();
> >        });
> >      })
> > +
> > +    it('is optional with a promise', function() {
>
> I'm using Mocha's Promise support here:
> https://mochajs.org/#working-with-promises
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/visionmedia/superagent/pull/925/files#r55382858>.
>